flask5/hotel/fields_check.py

```diff
@@ -0,0 +1,20 @@
+from flask_restful import abort
+
+
+def check_if_all_parameters_is_filled(args):
+    if not all(args.values()):
+        abort(400, error_code=400,
```

Comment:
do not use here flask functions, just return message. Where you call these functions you check if message then abort